lib/newrelic_riak/riak_client.rb

```diff
@@ -12,66 +13,84 @@
   end
 
   executes do
-    backend_tracers = proc do
-      add_method_tracer :ping, 'Database/Riak/ping'
 
-      add_method_tracer :list_buckets, 'Database/Riak/list_buckets'
-      add_method_tracer :get_bucket_props, 'Database/Riak/get_bucket_props'
-      add_method_tracer :set_bucket_props, 'Database/Riak/set_bucket_props'
+    # Make this bad boy re-usable
+    backend_tracers = Proc.new do |klass|
+      NewRelic::Agent::Datastores.trace klass, :ping
```

Comment:
It depends on what we want to do. From the docs:

```
An optional operation name can be included as the final parameter, otherwise the method name is used to represent the operation in metrics.
```

We used to name this something scoped to the client, then the method. I imagine the default behavior will be similar under the new code, but we could put the old paths back in... just assumed we should take them out for now to see if we still need them / what we actually need here.